pkg/e2e/osd/dedicatedadmin.go

```diff
@@ -70,5 +75,135 @@ var _ = ginkgo.Describe(dedicatedAdminTestName, func() {
+			Expect(err).NotTo(HaveOccurred())
+			daGroup.Users = append(daGroup.Users, "dummy-user@redhat.com")
+			daGroup, err = h.User().UserV1().Groups().Update(context.TODO(), daGroup, metav1.UpdateOptions{})
+			Expect(err).NotTo(HaveOccurred())
+			h.Impersonate(rest.ImpersonationConfig{
+				UserName: "dummy-admin@redhat.com",
+				Groups: []string{
+					"dedicated-admins",
+				},
+			})
+			defer func() {
+				h.Impersonate(rest.ImpersonationConfig{})
+			}()
+
+			_, err = createRolebinding("dummy-user@redhat.com", "admin", h)
+			Expect(err).NotTo(HaveOccurred())
+
+		}, float64(viper.GetFloat64(config.Tests.PollingTimeout)))
+
+		// can create edit rolebinding
+		ginkgo.It("can create edit rolebinding", func() {
+
+			daGroup, err := h.User().UserV1().Groups().Get(context.TODO(), "dedicated-admins", metav1.GetOptions{})
+			Expect(err).NotTo(HaveOccurred())
+			daGroup.Users = append(daGroup.Users, "dummy-user@redhat.com")
+			daGroup, err = h.User().UserV1().Groups().Update(context.TODO(), daGroup, metav1.UpdateOptions{})
+			Expect(err).NotTo(HaveOccurred())
+			h.Impersonate(rest.ImpersonationConfig{
+				UserName: "dummy-admin@redhat.com",
+				Groups: []string{
+					"dedicated-admins",
+				},
+			})
+			defer func() {
+				h.Impersonate(rest.ImpersonationConfig{})
+			}()
+
+			_, err = createRolebinding("dummy-user@redhat.com", "edit", h)
+			Expect(err).NotTo(HaveOccurred())
+
+		}, float64(viper.GetFloat64(config.Tests.PollingTimeout)))
+
+		// sa can create edit rolebinding
+		ginkgo.It("sa can create edit rolebinding", func() {
+
+			daGroup, err := h.User().UserV1().Groups().Get(context.TODO(), "dedicated-admins", metav1.GetOptions{})
+			Expect(err).NotTo(HaveOccurred())
+			daGroup.Users = append(daGroup.Users, "dummy-user@redhat.com")
+			daGroup, err = h.User().UserV1().Groups().Update(context.TODO(), daGroup, metav1.UpdateOptions{})
+			Expect(err).NotTo(HaveOccurred())
+
+			h.SetServiceAccount("system:serviceaccount:%s:dedicated-admin-project")
+
+			_, err = createRolebinding("dummy-user@redhat.com", "edit", h)
+			Expect(err).NotTo(HaveOccurred())
+
+		}, float64(viper.GetFloat64(config.Tests.PollingTimeout)))
+
+		// sa can create admin rolebinding
+		ginkgo.It("sa can create admin rolebinding", func() {
+
+			daGroup, err := h.User().UserV1().Groups().Get(context.TODO(), "dedicated-admins", metav1.GetOptions{})
+			Expect(err).NotTo(HaveOccurred())
+			daGroup.Users = append(daGroup.Users, "dummy-user@redhat.com")
+			daGroup, err = h.User().UserV1().Groups().Update(context.TODO(), daGroup, metav1.UpdateOptions{})
+			Expect(err).NotTo(HaveOccurred())
+
+			h.SetServiceAccount("system:serviceaccount:%s:dedicated-admin-project")
+
+			_, err = createRolebinding("dummy-user@redhat.com", "admin", h)
+			Expect(err).NotTo(HaveOccurred())
+
+		}, float64(viper.GetFloat64(config.Tests.PollingTimeout)))
 	})
 })
+
+func createNamespace(namespace string, h *helper.H) (*v1.Namespace, error) {
```

Comment:
I saw that function previously on namespace webhook and pod webhook aswell. Might not be in this PR but that can be exported into _util_  for wider usage and clearing out duplications

Reply:
removed func as I realized this is not needed.